recipe/meta.yaml

```diff
@@ -1,123 +1,206 @@
-{% set name = "cuda-python" %}
-{% set version = "12.6.2" %}
+{% set version = "0.1.1" %}
 {% set major_version = version.split(".")[0]|int %}
 {% set minor_version = version.split(".")[1]|int %}
 {% set patch_version = version.split(".")[2]|int %}
+{% set pkg_ver = major_version ~ "." ~ minor_version ~ "." ~ patch_version %}
+{% if major_version != 0 %}
+  {% set building_bindings = true %}
+  {% set building_core = false %}
+  {% set tag = "v" ~ pkg_ver %}
+  {% set number = 2 %}
+{% else %}
+  {% set building_bindings = false %}
+  {% set building_core = true %}
+  {% set tag = "cuda-core-v" ~ pkg_ver %}
+  {% set number = 0 %}
+{% endif %}
 {% set target_name = "x86_64-linux" %}  # [linux64]
 {% set target_name = "sbsa-linux" %}  # [aarch64]
 {% set target_name = "x64" %}  # [win64]
+{% set subdir = "" %}
```

Comment:
For some reason conda-smithy is not happy without a default value (jinja parsing failed with "subdir undefined".